src/screens/Notifications/AddNotificationEmailScreen.tsx

```diff
@@ -120,7 +120,10 @@ class AddNotificationEmailScreen extends PureComponent<Props, State> {
     const { onSkipSuccess } = route.params;
 
     createNotificationEmail('', {
-      onSuccess: onSkipSuccess,
+      onSuccess: () => {
+        this.props.navigation.popToTop();
```

Comment:
imho `this.props.navigation.popToTop();` should be added to `onSkipSuccess`  so it would remain flexible for every usage

Reply:
You should clear this screen from the stack every time so by moving it to `onSkipSuccess` you give the flexibility of accidentally not adding it.